.github/workflows/minimal-test.yml

```diff
@@ -7,22 +7,40 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.7, 3.8, 3.9]
-        os: [ubuntu-18.04]
+        python-version: ['3.8', '3.9', '3.10']
         flask-version: [latest]
+        os: [ubuntu-18.04]
     runs-on: ${{ matrix.os }}
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
+      - uses: actions/checkout@v3
```

Comment:
GitHub was encouraging us to update to the most recent action versions, seems harmless